### DISCUSSION TITLE
xccm for density binning...

### DISCUSSION THREAD

**Discussion by jklymak** [xgcm/xgcm#588] [Q&A]
Hi,  

I'm sure I'll figure it out, but maybe someone has a quick guide.  I have mitgcm output slices with density and tracers on fixed z-levels.  I'd like to bin in density space, both for the mean values in each bin, and for the volume of water in each bin.  This could be over the whole domain, or it might be nice to make x-pdens pooler plots of the distributions.

I think I'm close with `grid.transform(... method='conservative')`, but I'm not really understanding the units of what comes out of that.

**Comment by jbusecke** [comment]
Seems to be a duplicate of https://github.com/xgcm/xgcm/discussions/589 ?

**Reply by jklymak** [reply]
Yes somehow I must have hit the button twice or refreshed when I shouldn't have.  Sorry!
